docs/dev/Release.md

```diff
@@ -0,0 +1,365 @@
+     release.
+    
+        Example:
+        ```
+        Note for the committers:
+        until the release is not over and Drill version is not changed to 1.17.0-SNAPSHOT, please do not push any
+        changes into Drill master.
+        ```
+         
+2. ## Setup environment:
+    1. ### GPG key:
+        You will need a GPG key set up. This key is needed for signing the release.
+        1. Read [Apache docs for release signing](http://www.apache.org/dev/release-signing.html).
+        2. Install the `gpg2` package on Ubuntu, or `gnupg` on MacOS.
+        3. Generate GPG key using the **Apache email address** if it wasn't done before:
+            1. Run `gpg --gen-key` and follow the instructions.
+            2. Remember your passphrase.
+            3. Save your key pair:
+                ```
+               gpg --output mygpgkey_pub.gpg --armor --export MY_KEY_ID
+               gpg --output mygpgkey_sec.gpg --armor --export-secret-key MY_KEY_ID
+               ```
+        4. Or import existing GPG key pair if required:
+            ```
+            gpg --import mygpgkey_pub.gpg
+            gpg --allow-secret-key-import --import mygpgkey_sec.gpg
+            ```
+        5. Have another committer signed your key (add to the trust chain).
+            Apache advises to do it at 
+            [key signing parties](https://www.apache.org/dev/release-signing.html#key-signing-party).
+        6. Make sure default key is the key generated using the Apache email.
+        7. Publish your public key to a public server (e.g. http://pgp.surfnet.nl or http://pgp.mit.edu)
+            using Web-UIs or CLI (for example using the following command:
+            `gpg --keyserver hkp://pool.sks-keyservers.net --trust-model always --send-keys`)
+        8. Add your key signature to the `KEYS` file in the Drill sources repo.
+            Instruction on how to create the key signature is embedded at the beginning of the KEYS file.
+        9. Add your key signature to the `KEYS` file in `https://dist.apache.org` (PMC permissions required for this):
+            - `svn co https://dist.apache.org/repos/dist/release/drill ~/src/release/drill-dist`
+            - Update `KEYS` file with required changes
+            - Commit changes: `svn ci -m "Add FirstName LastName's key to KEYS file"`
+    2. ### Setting up LDAP credentials
+        1. Go to http://id.apache.org and log in using your Apache login and password.
+        2. Scroll to the bottom to the 'SSH Key' entry and add the SSH key from your `~/.ssh/id_rsa.pub` file.
+            If you don't have the SSH key, you should generate it by running `ssh-keygen`.
+        3. Note that you can add more than 1 SSH key corresponding to multiple machines.
+        4. Enter your Apache password and submit changes.
+        5. Verify that you can do an sftp to the apache server by running the following: `sftp <username>@home.apache.org`
+
+    3. ### SVN
```

Comment:
`9. Add your key signature to the `KEYS` already requires SVN looks like having this step after point 9 is strange
How to setup up SVN access?